cluster-up/cluster/kind/common.sh

```diff
@@ -152,15 +181,10 @@ function setup_kind() {
         docker exec $node /bin/sh -c "curl -L https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-linux-amd64-v0.8.5.tgz | tar xz -C /opt/cni/bin"
     done
 
-    echo "ipv6 cni: $IPV6_CNI"
-    if [ -z ${IPV6_CNI+x} ]; then
-        echo "no ipv6, safe to install calico"
-        _kubectl apply -f $KIND_MANIFESTS_DIR/kube-calico.yaml
-    else
-        echo "ipv6 enabled, using kindnet"
-        # currently kind does not fully support ipv6 or ipv6-DualStack,
-        # when using diffrent CNI's.
-    fi
+    # Deploy CNI
+    cni_patched_manifest="/tmp/calico.yaml"
+    _patch_calico_manifest "$KIND_MANIFESTS_DIR/kube-calico.yaml.in" "$KIND_MANIFESTS_DIR/kube-calico.diff.in" "$cni_patched_manifest"
+    _kubectl apply -f "$cni_patched_manifest"
```

Comment:
I don't know if it's worth it but you can use stdin with `kubectl apply -f -` so no need for /tmp/calico.yaml